ibm/service/secretsmanager/resource_ibm_sm_private_certificate_configuration_intermediate_ca.go

```diff
@@ -406,15 +412,25 @@ func resourceIbmSmPrivateCertificateConfigurationIntermediateCARead(context cont
 	if err = d.Set("max_ttl_seconds", flex.IntValue(configuration.MaxTtlSeconds)); err != nil {
 		return diag.FromErr(fmt.Errorf("Error setting max_ttl_seconds: %s", err))
 	}
+	if d.Get("max_ttl") == nil || d.Get("max_ttl") == "" {
```

Comment:
This code is needed when the resource is imported into Terraform. The response to the GET request of the secrets manager API does not include the value of `max_ttl`. That's a problem because this is a required field.Normally after import, the user can copy the values of the required fields from the TF state, but because this field isn't returned in the response, it wouldn't exist in the TF state. The fix here is to add the field `max_ttl` in the provider read function, and the value is taken from the alternate field `max_ttl_seconds`. This guarantees that after import, the TF state will have a value for `max_ttl`.